assets/xml/textures/nes_font_static.xml

```diff
@@ -14,130 +14,131 @@
+        <Texture Name="gMsgCharLatinSmallLetterEWithCircumflexTex" OutName="msg_char_latin_small_letter_e_with_circumflex" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLatinSmallLetterEWithDiaeresisTex" OutName="msg_char_latin_small_letter_e_with_diaeresis" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLatinSmallLetterIWithDiaeresisTex" OutName="msg_char_latin_small_letter_i_with_diaeresis" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLatinSmallLetterOWithCircumflexTex" OutName="msg_char_latin_small_letter_o_with_circumflex" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLatinSmallLetterOWithDiaeresisTex" OutName="msg_char_latin_small_letter_o_with_diaeresis" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLatinSmallLetterUWithGraveTex" OutName="msg_char_latin_small_letter_u_with_grave" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLatinSmallLetterUWithCircumflexTex" OutName="msg_char_latin_small_letter_u_with_circumflex" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharLatinSmallLetterUWithDiaeresisTex" OutName="msg_char_latin_small_letter_u_with_diaeresis" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharButtonATex" OutName="msg_char_button_a" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharButtonBTex" OutName="msg_char_button_b" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharButtonCTex" OutName="msg_char_button_c" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharButtonLTex" OutName="msg_char_button_l" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharButtonRTex" OutName="msg_char_button_r" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharButtonZTex" OutName="msg_char_button_z" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharButtonCUpTex" OutName="msg_char_button_c_up" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharButtonCDownTex" OutName="msg_char_button_c_down" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharButtonCLeftTex" OutName="msg_char_button_c_left" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharButtonCRightTex" OutName="msg_char_button_c_right" Format="i4" Width="16" Height="16"/>
         <Texture Name="gMsgCharZTargetSignTex" OutName="msg_char_z_target_sign" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharAnalogStickTex" OutName="msg_char_analog_stick" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharDPadTex" OutName="msg_char_dpad" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharControlStickTex" OutName="msg_char_control_stick" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharPlusControlPadTex" OutName="msg_char_plus_control_pad" Format="i4" Width="16" Height="16"/>
```

Comment:
we use dpad elsewhere in the codebase (input macros) and its what a majority of people call it, dont think we should change that

Reply:
agreed

Reply:
Okay, for DPad (although worth noting that the in-game text explicitly calls it Control Pad: 
![image](https://user-images.githubusercontent.com/73679967/113480518-7c7f9380-948c-11eb-9b1f-05862069a5ba.png)
 ). `ControlStick` is used in `gameplay_keep` already, and we don't have to worry about regional spelling differences, so I'd rather keep that one.

Reply:
Instruction Manuals also call it the Control Pad
![image](https://user-images.githubusercontent.com/7389110/113480767-f49a8900-948d-11eb-9db9-0de051230abb.png)

Reply:
Indeed, it's been called that by Nintendo from NES: https://www.manualslib.com/manual/1010406/Nintendo-Nintendo-Entertainment-System.html?page=4#manual to Wii U: https://www.nintendo.com/consumer/downloads/manual-nintendo-wiiu-operations-english.pdf .

Reply:
Don't forget the Switch.
https://cdn02.nintendo-europe.com/media/downloads/support_1/nintendoswitch/NSwitch_Information_ProController_EUR.pdf
![image](https://user-images.githubusercontent.com/7416381/113481003-93fb5400-946d-11eb-88cc-525bf77a6824.png)

Reply:
Yea, elliptic noted in the original comment that its from the manual. No one calls it that though, and will probably just be confused when seeing it lol

Reply:
What about ``...Control_D_Pad...`` as a compromise?

Reply:
no need imo when dpad is already well established

Reply:
Control Pad is the official name for it though. Doesn't make sense to break from it just because there's a more popular English term.

Reply:
it makes perfect sense to, in the same way we dont use official symbol names when we have something thats better/known more commonly

Reply:
What official symbol name we aren't using?

Reply:
search `original name:` in the codebase

Reply:
There's a difference between symbol names in source and officially published terminology that appears in hundreds of different manuals and game dialog.

Reply:
Is it really comparable a "debug variable name" to the name of a button that is it actually in the game and in every game manual?
Variables names aren't something official imo, they aren't supposed to be seen by the end user.

Reply:
When 99% of people who see that name are gonna have to look it up to know what it means, it doesnt really matter how official it is or how many manuals it appears in, in my opinion. Have never heard it called Plus Control Pad ever

Reply:
we're discussing the name of the variable as its going to be seen in the code. It should be as recognizable as possible so the person reading the code knows instantly what its for

Reply:
Control Pad is hardly super confusing terminology, and D-pad only makes sense in English.

Reply:
Texture Name and Internal Name should be kept as close as possible just for sanity's sake